### ISSUE TITLE
Create DelegateWork task/meta task and design an UI to manually assign tasks

### ISSUE THREAD

**Issue by mokun** [mars-sim/mars-sim#915]
**Is your feature request related to a problem? Please describe.**
- Commander or a division chief does not have any significance in mars-sim.

**Describe the rationale**
- Automation is good and currently the settlement determines the needs and put those tasks into the Task backlog.
- However, if the player is the commander or a division chief, it should be his job to determine the priority of tasks and delegate those tasks.

**Describe the solution you desire to see**
- Give commander and chief some types of tasks that are impacting the settlement.
- Add a new task/meta task called **DelegateWork** for commander/chief to handle the selection of tasks for their subordinates to do
- Provide a checkbox for player to choose whether he would like to personally assign tasks or let the AI assign post them in the task backlog system.
- Provide an UI for player to handle the flow of tasks. He may drag and drop tasks from a common pool into a division pool for each division to pick up tasks.

**Specifications  (please complete the following information):**
 - pre 3.6.0 build 8180 
 
**Additional context**
- In future, we should assess how well a commander/chief is performing his job. However, this implies a task tracking system in place before we can say if they are high or low achievers.

**Comment by bevans2000** [comment]
How do you see this working? If the user is assigning tasks to Person/Robots then it will make it a very slow process.

**Comment by mokun** [comment]
>  If the user is assigning tasks to Person/Robots then it will make it a very slow process

The default is that the commander/chief will automatically do DelegateWork.

Player can choose to overrides it and set up his own template.

So the key is to create a template that allows player to customize those tasks and missions assignments.

A template for each division would place restrictions or permissions and specify modifier scores for either specific person or just roles to do certain tasks and missions.

**Comment by bevans2000** [comment]
My ideas on Governance could be expanded into this area for sure. The first step is to identify Tasks that can only be assigned by someone in charge and not chosen freely by a Worker.
Let's work on the delegate as a follow on.
